db/migrate/20141010133702_create_posts.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.string :title
       t.text :content_md
       t.text :content_html
+      t.text :price
```

Comment:
that's one of the initial migration. You don't change existing migrations, you generate new ones. It is a Rails thing, you can read about it on
here -> http://guides.rubyonrails.org/active_record_basics.html
and 
here -> http://guides.rubyonrails.org/active_record_migrations.html